packages/form-js-viewer/src/render/hooks/useOptionsAsync.js

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from 'preact/hooks';
```

Comment:
We didn't export this hook, right?

Reply:
Oh I didn't think about it but no we didn't. Luuuckyyy.

Reply:
People might ask about it once custom components are released. Once we do it, we have to be more careful about renaming 👍